README.windows.md

```diff
@@ -183,6 +183,11 @@ versions of the MinGW-w64 compilers available through Cygwin's package manager.
     or [64 bit](http://cygwin.com/setup-x86_64.exe). Note, that you can compile
     either 32 or 64 bit Julia from either 32 or 64 bit Cygwin. 64 bit Cygwin
     has a slightly smaller but often more up-to-date selection of packages.
+    
+    Advanced: you may skip steps 2-4 by running:
+    
+        setup-x86_64.exe -q -P cmake,gcc-g++,git,make,patch,curl,m4,python,p7zip,mingw64-i686-gcc-g++,mingw64-i686-gcc-fortran,mingw64-x86_64-gcc-g++,mingw64-x86_64-gcc-fortran
+        :: add -s <url> if you want to override the default download site
```

Comment:
if I drop that argument from my vagrantfile it hangs, possibly waiting for human input which isn't that useful for a script